docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -589,6 +589,17 @@ table in the metastore, using its existing transaction logs and data files::
 To prevent unauthorized users from accessing data, this procedure is disabled by default.
 The procedure is enabled only when ``delta.register-table-procedure.enabled`` is set to ``true``.
 
+.. _delta-lake-unregister-table:
+
+Unregister table
```

Comment:
![image](https://user-images.githubusercontent.com/57260417/215085246-8a4e1de4-d011-44cf-9583-e12cd03836d2.png)


Change to 

```
Unregister table
^^^^^^^^^^^^
```

Reply:
Good catch, fixed.